lib/livebook.ex

```diff
@@ -199,17 +199,22 @@ defmodule Livebook do
       config :livebook, :cacertfile, cacertfile
     end
 
-    if rewrite_on = Livebook.Config.rewrite_on!("LIVEBOOK_PROXY_HEADERS") do
-      config :livebook, :rewrite_on, rewrite_on
-    end
+    config :livebook, :rewrite_on, Livebook.Config.rewrite_on!("LIVEBOOK_PROXY_HEADERS")
 
     config :livebook,
            :cookie,
            Livebook.Config.cookie!("LIVEBOOK_COOKIE") ||
              Livebook.Config.cookie!("RELEASE_COOKIE") ||
              Livebook.Utils.random_cookie()
 
-    if node = Livebook.Config.node!("LIVEBOOK_NODE", "LIVEBOOK_DISTRIBUTION") do
+    # TODO: remove in v1.0
+    if System.get_env("LIVEBOOK_DISTRIBUTION") == "sname" do
+      Livebook.Config.abort!(
+        "Found LIVEBOOK_DISTRIBUTION=sname, but short names are no longer supported."
+      )
```

Comment:
Just warn and say they are ignored? This way it still boots and it should be less disruptive.